### PR TITLE
notification indication, contributors fixed again, changed modules fixed

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: packages/app/src/app/overmind/effects/api/index.ts [hunk 1]
```diff
@@ -62,12 +62,13 @@ export default {
...
     const sandbox = await api.get<Sandbox>(`/sandboxes/${id}`);
 
-    // We need to add savedCode property to track it
+    // We need to add client side properties for tracking
     return {
       ...sandbox,
       modules: sandbox.modules.map(module => ({
         ...module,
         savedCode: null,
+        isNotSynced: false,
```
Comment (CompuIves) [inline packages/app/src/app/overmind/effects/api/index.ts]: Interesting, where do we use this?

File: packages/app/src/app/overmind/effects/live/clients.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ class CodeSandboxOTClient extends Client {
   }
 
   sendOperation(revision, operation) {
-    this.onSendOperation(revision, operation);
+    this.onSendOperation(revision, operationToElixir(operation.toJSON()));
```
Comment (CompuIves) [inline packages/app/src/app/overmind/effects/live/clients.ts]: I wonder how `master` currently works without this 🤔. But nice to see it fixed now!

Comment (christianalfoni) [reply packages/app/src/app/overmind/effects/live/clients.ts]: Was it not opposite? I had merge conflict. I chose the wrong thing, it is handled up above, 2 sec

File: packages/app/src/app/overmind/effects/api/index.ts (hunk 1 above)
Comment (christianalfoni) [reply packages/app/src/app/overmind/effects/api/index.ts]: It is being used in FilePath, ImageViewer and FuzzySearch... maybe related to iPad only?

Comment (CompuIves) [reply packages/app/src/app/overmind/effects/api/index.ts]: Oh wow, do we ever change this property? Might be a legacy thing from the redux days.

Comment (christianalfoni) [reply packages/app/src/app/overmind/effects/api/index.ts]: We should track synced state on modules anyways, and make changedModuleshortids a derived, so this is a good reminder :)

File: packages/app/src/app/overmind/effects/live/clients.ts (hunk 2 above)
Comment (christianalfoni) [reply packages/app/src/app/overmind/effects/live/clients.ts]: This is fixed now

**Pull request merged**